test/ApplicationContainer.test.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable */
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { axe, toHaveNoViolations } from "jest-axe";
+import "@testing-library/jest-dom/extend-expect";
+
+import ApplicationContainer from "../src/components/ApplicationContainer";
+
+expect.extend(toHaveNoViolations);
+
+describe("ApplicationContainer", () => {
+  test("passes axe", async () => {
+    const { container } = render(<ApplicationContainer />);
+
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  test("renders with a header and footer", () => {
+    const { container } = render(<ApplicationContainer />);
+
+    expect(container.querySelector("header")).toBeInTheDocument();
+    expect(container.querySelector("footer")).toBeInTheDocument();
+  });
+
+  test("renders a banner", () => {
+    render(<ApplicationContainer />);
```

Comment:
Get rid of some of the extra blank lines throughout this file?

Reply:
Like which ones?... The linter/prettier script runs through the files when committing so everything should be consistent with the plugin preferences.

Reply:
There are a few that look slightly off to me, but it's not worth arguing with the linter; it's fine!